_projects/community-share.md

```diff
@@ -4,7 +4,7 @@ date: 2015-06-01
 link: https://github.com/communityshare/communityshare
 intro: Come help with
 image: /assets/images/communityshare.png
-lead: Josh Schachter & Ben Reynwar
```

Comment:
I have taken Josh off of the list here because I thought that he no longer was part of the project. If this is wrong, we can keep him in there.